.env.example

```diff
@@ -8,4 +8,12 @@ MAIL_PORT=
 MAIL_USER=
 MAIL_PASSWORD=
 MAIL_SENDER=
-SECRET=
\ No newline at end of file
+SECRET=
+GOOGLE_OAUTH_CLIENT_ID=
```

Comment:
This .env file is starting to get a bit complex, but I'm not sure what the best solution is. :thinking:

Reply:
Personally, no idea. It is getting ridiculous and I can't think of anything else to be honest.